### PR TITLE
Move calibration from commander to event framework

### PR DESCRIPTION
**Pull request by sugnanprabhu**
This patch moves all calibration routines out of commander to events framework and removes commander_low_prio_loop.

Includes following set of changes.
- On receiving calibration request
     i. commander does some minimal handling and move the arming state to ARMING_STATE_INIT.
     ii. event framework waits until the arming state is moved to ARMING_STATE_INIT before timeout of 2secs.
- event framework spawns a px4 task to handle the calibration and make sure only one calibration is active.
- event framework sends a calibration_status orb message after the completion of the calibration.
- Commander will move the arming state back to ARMING_STATE_STANDBY once it receives calibration_status.

This is one of the part of task mentioned in #8014

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Cool, thanks! @bkueng @dagar More RAM!

**Comment by dagar** [comment]
Looks great! I resolved the conflicts, deleted some remaining low priority thread configuration, and rebased on px4 master. I'd like to get this in right after the v1.7.0 release.

File: src/modules/events/calibration/airspeed_calibration.cpp [hunk 1]
```diff
@@ -237,9 +242,10 @@ int do_airspeed_calibration(orb_advert_t *mavlink_log_pub)
 			}
 
 			if (calibration_counter % 500 == 0) {
-				calibration_log_info(mavlink_log_pub, "[cal] Create air pressure! (got %d, wanted: 50 Pa)", (int)diff_pres.differential_pressure_filtered_pa);
-				tune_neutral(true);
```
Comment (dagar) [inline src/modules/events/calibration/airspeed_calibration.cpp]: TODO - review all calibration tunes

File: src/modules/events/calibration/airspeed_calibration.cpp [hunk 2]
```diff
@@ -257,7 +263,6 @@ int do_airspeed_calibration(orb_advert_t *mavlink_log_pub)
 	calibration_log_info(mavlink_log_pub, CAL_QGC_PROGRESS_MSG, 100);
 
 	calibration_log_info(mavlink_log_pub, CAL_QGC_DONE_MSG, sensor_name);
-	tune_neutral(true);
```
Comment (dagar) [inline src/modules/events/calibration/airspeed_calibration.cpp]: TODO - review calibration tunes

**Comment by dagar** [comment]
TODO for @dagar
 - review commander stack requirements
 - review events stack requirements
 - tune usage (move to uORB?)
 - test everywhere (monitor memory usage)

**Comment by dagar** [comment]
@simonegu @LorenzMeier can we sync up this week to discuss plans for the tone_alarm uORB architecture?
https://github.com/PX4/Firmware/pull/7316

I'd strongly prefer getting tone usage via uORB in place first so that the calibration code isn't directly calling into the commander library in this PR. In fact we should also update the build system so that modules don't have free access to other modules like that (https://github.com/PX4/Firmware/pull/8378).

**Comment by LorenzMeier** [comment]
@sugnanprabhu Do you have time this week to rebase this? Ideally this would include porting the remaining parts of Commander into the (new) class architecture.

**Comment by LorenzMeier** [comment]
I've rebased this here:
https://github.com/PX4/Firmware/pull/8630

@sugnanprabhu Could you review if I did the rebase correctly? If not, please rebase yourself and I'll delete my branch.

**Pull request closed without merging**